app/main.py

```diff
@@ -1,3 +1,78 @@
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float | int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | float) -> "Distance":
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        elif isinstance(other, (int, float)):
+            return Distance(self.km + other)
+
+    def __iadd__(self, other: "Distance") -> "Distance":
+        if isinstance(other, Distance):
+            self.km += other.km
+        elif isinstance(other, (int, float)):
+            self.km += other
+        else:
+            raise TypeError("Unsupported operand type for +=")
+        return self
+
+    def __mul__(self, other: "Distance") -> "Distance":
+        if isinstance(other, (int, float)):
+            return Distance(self.km * other)
+        else:
+            raise TypeError("Unsupported operand type for *")
+
+    def __truediv__(self, other: "Distance") -> "Distance":
+        if isinstance(other, (int, float)):
+            result = self.km / other
+            return Distance(round(result, 2))
+        else:
+            raise TypeError("Unsupported operand type for /")
+
+    def __lt__(self, other: ["Distance", int, float]) -> bool:
+        if isinstance(other, Distance):
+            return self.km < other.km
+        elif isinstance(other, (int, float)):
+            return self.km < other
+        else:
+            raise TypeError("Unsupported operand type for <")
```

Comment:
```suggestion
        return self.km < other
```
Fix in all places! This check is redundant